x-pack/plugins/lens/public/native_renderer/native_renderer.test.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { render } from 'react-dom';
+import { NativeRenderer } from './native_renderer';
+import { act } from 'react-dom/test-utils';
```

Comment:
I wasn't familiar with `act`, so I read the docs, and they say this is not necessary if you use: `react-test-renderer` which renders to JSON: https://reactjs.org/docs/test-renderer.html

I assume we can't use this because we actually need the JSDOM to exist and be used internally?

Reply:
Yes, I wanted to test whether this actually translates correctly the the underlying DOM operations as it's such a low level concern - this is why I used the actual DOM render calls for this suite.